includes/free-trial/class-wc-calypso-bridge-free-trial-plugins-screen.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * WC Calypso Bridge Free Trial Plugins Screen - Landing page.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   2.2.8
+ * @version 2.2.8
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Calypso_Bridge_Free_Trial_Plugins_Screen Class.
+ */
+class WC_Calypso_Bridge_Free_Trial_Plugins_Screen {
+
+	/**
+	 * Class instance.
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance.
+	 */
+	final public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	protected function __construct() {
+		// Only for free trials.
+		if ( ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
+
+		$this->init();
+	}
+
+	/**
+	 * Initialize hooks.
+	 */
+	protected function init() {
+		add_action( 'admin_menu', array($this, 'add_menu_page'));
+		add_filter( 'admin_body_class', function( $classes ) {
+				$screen = get_current_screen();
+				if ( $screen && 'toplevel_page_plugins-upgrade' === $screen->id ) {
+					$classes .= 'woocommerce_page_wc-bridge-landing-page woocommerce_page_wc-plugins-landing-page';
```

Comment:
Should this have leading or trailing spaces, to make sure spacing is always maintained?

Reply:
Good catch, @beaulebens! I've added leading and trailing spaces along with another instance of this filter in 883dfafcaa989bad6a2fe53d4f9df98059f91b17